### PR TITLE
Feature/video graphics in media library

### PR DESCRIPTION
**Pull request by pjosh**
* Video thumbnails in media library index.
* Iframe embed in video show page.
* Collections & Graphics support in front.

### CODE REVIEW

File: app/views/media_library/show.html.erb [hunk 1]
```diff
@@ -8,9 +8,21 @@
                 when MediaContent::TYPE_PHOTO
                   is_vertical = @media_content.photo_sizes.where(label: PhotoSize::LARGE).first.is_vertical
                   @media_content.photo_sizes.where(label: PhotoSize::LARGE).first.try(:url)
+                when MediaContent::TYPE_COLLECTION
+                  photo = @media_content.graphics.first
+                  is_vertical = photo.photo_sizes.where(label: PhotoSize::LARGE).first.is_vertical
+                  photo && photo.photo_sizes.where(label: PhotoSize::LARGE).first.try(:url)
+                when MediaContent::TYPE_GRAPHIC
```
Comment (pjosh) [inline app/views/media_library/show.html.erb]: @simaob how can I put the `TYPE_PHOTO` & `TYPE_GRAPHIC` in the same condition? I was try with `MediaContent::TYPE_PHOTO || MediaContent::TYPE_GRAPHIC` but doesn't work.

Comment (simaob) [inline app/views/media_library/show.html.erb]: with a comma:

`when MediaContent::TYPE_GRAPHIC, MediaContent::TYPE_PHOTO`

Comment (pjosh) [inline app/views/media_library/show.html.erb]: :dizzy_face::gun: thx @simaob

**Comment by simaob** [comment]
Some of the videos might be on youtube, we should allow for both.

**Pull request merged**